sources/packages/backend/apps/api/src/auth/auth.module.ts

```diff
@@ -87,6 +93,8 @@ const jwtModule = JwtModule.register({
       provide: APP_GUARD,
       useClass: SINValidationGuard,
     },
+    { provide: APP_GUARD, useClass: InstitutionBCPublicGuard },
```

Comment:
suggestion, can we follow the other format 
![image](https://user-images.githubusercontent.com/77353155/234954623-4bd4d916-7152-49da-a160-b093713824a5.png)